lib/build.sh

```diff
@@ -172,3 +172,10 @@ prune_devdependencies() {
   npm prune --userconfig "$build_dir/.npmrc" 2>&1
   log_info "Successfully pruned devdependencies!"
 }
+
+warn_prebuilt_modules() {
+  local build_dir=$1
+  if [ -e "$build_dir/node_modules" ]; then
+    log_info "node_modules checked into source control" "https://blog.heroku.com/node-habits-2016#9-only-git-the-important-bits"
```

Comment:
I'd rather not link to this blog post since it's so old... I think this article is better: https://devcenter.heroku.com/articles/node-best-practices#only-git-the-important-bits

Reply:
Ok! Should I change it here (https://github.com/heroku/heroku-buildpack-nodejs/blob/9898b875f45639d9fe0fd6959f42aea5214504db/lib/failure.sh#L633) as well?

Reply:
Yep! That sounds good... thanks!